external/CMakeLists.txt

```diff
@@ -2,7 +2,10 @@ include(ExternalProject)
 
 if(CMAKE_CROSSCOMPILING)
   set(CONFIGURE_FLAGS
-    --host=${CMAKE_SYSTEM_PROCESSOR}-none-eabi --build=${CMAKE_HOST_SYSTEM_PROCESSOR}-linux-gnu)
+    --host=${CMAKE_SYSTEM_PROCESSOR}-none-eabi
+    --build=${CMAKE_HOST_SYSTEM_PROCESSOR}-linux-gnu
+    "$<$<BOOL:${CMAKE_C_COMPILER_LAUNCHER}>:CC=${CMAKE_C_COMPILER_LAUNCHER} ${CMAKE_C_COMPILER}>"
```

Comment:
doesn't cmake automatically prepend/use the CMAKE_C_COMPILER_LAUNCHER? why is this line necessary?

Reply:
good to know - pls add a comment :smile:

Reply:
nevermind, i thought we were in a different CMakeLists.txt. It's in external/, so that's clear enough.